packages/js/product-editor/src/hooks/test/use-error-handler.test.ts

```diff
@@ -36,7 +36,8 @@ jest.mock( '@wordpress/data', () => ( {
 
 jest.mock( '../use-blocks-helper', () => ( {
 	useBlocksHelper: jest.fn().mockReturnValue( {
-		getParentTabId: jest.fn( ( context ) => context ),
+		getParentTabId: jest.fn( () => 'inventory' ),
+		getParentTabIdByBlockName: jest.fn( () => 'inventory' ),
```

Comment:
I added a test for the case when `getParentTabIdByBlockName` returns `null` in the commits [e2b7849](https://github.com/woocommerce/woocommerce/pull/49729/commits/e2b784969a656b872ddb0f6134b10a1931840e85) and [eaec213](https://github.com/woocommerce/woocommerce/pull/49729/commits/eaec2137a79a6fecddb39cee45afe35cd35b58c5)

Reply:
I appreciate you adding a use case for `getParentTabIdByBlockName`, but I would appreciate some more thought around the actual tests as it would of also been nice to add a test case for `getParentTabId` returning `null` as well.
It's ok to spend some extra time on making sure the tests are correct and all test cases are considered.